README.md

```diff
@@ -329,6 +330,7 @@ Alongside the built assets a metadata file will be created with info about the b
     "configure_arguments": <output of bash configure>
```

Comment:
Not sure I like this being in red - wonder if there's another way we can format it?

Reply:
I'll stick in a string so it doesn't get parsed as JSON operators
```json
"configure_arguments": "<output of bash configure>"
```